agile.tex

```diff
@@ -48,19 +50,18 @@ \subsection{Long Term Planning}
 Planning packages are defined at the fourth level of the Work Breakdown Structure (WBS).
 
 All WBS elements are related to the set of Data Management Products embodied in the System Design \cite{LDM-148}.
-Each product has a product owner to guide the agile development, this is frequently one of the Data Management Scientists.
+Each product has a \emph{product owner} to guide the agile development, this is frequently one of the Data Management Scientists.
 
-During the cycle planning process  effort is drawn from the budget embodied in the planning packages to generate the cycle plan, described in terms of epics in Jira.
-Each epic itself has a particular budget.
+During the planning of each 6 month cycle, effort is drawn from the budget allocated to the planning packages to generate the cycle plan, described in terms of epics in Jira.
+Each epic itself has a particular budget with story points being associated with a specific WBS.
```

Comment:
This is better but I still find it unclear. I think you are saying two related things:
- Each epic is associated with a WBS element (surely not a WBS?)
- Each epic has a budgeted amount of effort that is drawn from the budget of that WBS element

Consider something like "Each epic is associated with a specific WBS element and its effort (expressed as <i>story points</i>) is drawn from that element's budget".

Refinements:
- If the term "story points" is not used elsewhere you could remove the parenthetical comment.
- "effort" could be expanded, but I find "amount of effort" clumsy; perhaps "budgeted effort" (which adds stutter) or perhaps the parenthetical comment helps?